test/e2e-tests.sh

```diff
@@ -105,6 +105,11 @@ go_test_e2e -timeout=20m -parallel=300 ./test/scale ${TEST_OPTIONS} || failed=1
 # Run HPA tests
 go_test_e2e -timeout=15m -tags=hpa ./test/e2e ${TEST_OPTIONS} || failed=1
 
+# Run emptyDir tests with alpha enabled avoiding any issues with the testing options guard above
+toggle_feature kubernetes.podspec-volumes-emptydir Enabled
+go_test_e2e -tags=emptydir ./test/e2e --enable-alpha ${TEST_OPTIONS} || failed=1
```

Comment:
We need to add emptydir test in `openshift/e2e-common.sh` actually. 
But this is not only `release-0.24.0` but also `release-next` branch also does not test it yet :sweat: